closure/goog/net/xhriopool.js

```diff
@@ -43,6 +49,13 @@ goog.net.XhrIoPool = function(opt_headers, opt_minCount, opt_maxCount) {
    */
   this.headers_ = opt_headers;
 
+  /**
+   * Add credentials to every request.
```

Comment:
More detail on what "withCredentials_" means would be useful here. See the associated docs in XhrIo for an example.